Open_Science_New.html

```diff
@@ -291,83 +346,80 @@ <h2>Standardization</h2>
 	  <section data-transition="concave">
             <img src="images/EEG_Signal.png">
           </section>
-	  
-	</section>		  
-	
-	<!-- Infrastructure-->
-	<section data-transition="convex">
-	  <section>
-	    <h2>Improved Infrastructure</h2>
-	    <span class="fragment"><p><font color="f27c30">Databases, NoSQL, APIs, Version Control, Provenance Capture</p></font></span>
+
+	</section>
+
+          <!-- Infrastructure Column 9-->
+          <section>
+              <section data-transition="fade" data-background="images/infrastructure.png">
+	      <h1 style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;"><font color="999900">Improved Infrastructure</font></h1>
 	  </section>
 
 	    <section>
 	    <h2>Connectivity and Data Access</h2>
-	    <img src="images/HPC_map.png">
+	    <a href="https://www.computecanada.ca"><img src="images/HPC_map.png"></a>
 	  </section>
 
 	    <section>
-	    <h2>HPCs</h2>
+	    <h5>High Productivity Computing Systems (HPCs)</h5>
 	    <img src="images/HPC_example.png" width="87%">
 	  </section>
-	
+
 	  <section>
 	    <h2>SQL versus NoSQL</h2>
 	    <img src="images/NoSQL_database.jpg" width="77%">
 	  </section>
-	
-	  <!-- Git and Github -->
-	  <section data-transition="fade">
-	    <table class="reveal">
-	      <h2>Git & Github</h2>
-	      <tr>
-		<td width="50%" style="vertical-align: middle">
-		  <img src="images/git.png">
-		</td>
-		<td width="50%" style="vertical-align: middle">
-		  <img src="images/Github.png">
-		</td>
-	    </tr></table>
-	  </section>
-	  
-	  <!-- Git Annex -->
-	  <section data-transition="fade">
-	    <img src="images/Git_annex.png">
-	  </section>
 
-	  <!-- Git Annex -->
-	  <section data-transition="fade">
-	    <img src="images/Datalad.png">
-	  </section>
+	  <!-- Git -->
+              <section>
```

Comment:
@christinerogers Another issue with fitting on one page - do you know a solution? Also to remove that weird border line? I consolidated all the git slides into this one slide so if there's no way to make it fit we can just re-separate them.

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/9/4